setup.py

```diff
@@ -1,58 +1,43 @@
 #!/usr/bin/env python
 
 import sys
-from setuptools import setup
-from setuptools.command.test import test as TestCommand
+from setuptools import setup, Command
 
-REQUIREMENTS = []
-
-TEST_REQUIREMENTS = [
-    "pytest>=2.7.2",
-]
-
-class PyTest(TestCommand):
+class PyTest(Command):
     user_options = []
 
     def initialize_options(self):
-        TestCommand.initialize_options(self)
-        self.pytest_args = []
+        pass
 
     def finalize_options(self):
-        TestCommand.finalize_options(self)
-        self.test_args = []
-        self.test_suite = True
+        pass
 
-    def run_tests(self):
+    def run(self):
         import pytest
-        errno = pytest.main(self.pytest_args)
-        sys.exit(errno)
+        sys.exit(pytest.main([]))
 
 setup(
     name='ish',
     version="0.0.0",
     description="Make Python more like more sensible languages.",
-    url="https://github.com/judy2k/fictionary",
+    url="https://github.com/judy2k/ish",
 
     author="Mark Smith",
     author_email="<mark.smith@practicalpoetry.co.uk>",
 
     classifiers=[
         "Development Status :: 4 - Beta",
-        "Environment :: Console",
-        "License :: OSI Approved :: MIT License",
-        "Natural Language :: English",
+        "License :: Public Domain",
```

Comment:
Note that there isn't a classifier for the Unlicense, however Public Domain is more accurate than MIT license.